### PR TITLE
Update reviews by semester

### PR DESCRIPTION
**Pull request by davish**
This PR adds utility functions and manage.py support for updating PCR ratings for a single semester, thanks to a new PCR endpoint!

### CODE REVIEW

File: review/management/commands/pcrbackup.py [hunk 1]
```diff
@@ -19,10 +20,14 @@ def handle(self, *args, **kwargs):
         else:
             answer = input('Output directory already exists. Overwrite existing contents? (y/N): ')
             if answer.lower() != 'y':
```
Comment (mureytasroc) [inline review/management/commands/pcrbackup.py]: This isn't something new so it's prob fine but how come you are allowing 'y' and 'Y'?  Isn't it usually convention to require y to be lowercase and N to be uppercase?

File: .gitignore [hunk 2]
```diff
@@ -14,4 +14,4 @@ PennCourses/settings/static/
 *.sqlite3
 *-test.json
 celerybeat-schedule
-pcr-backup
+pcr-backup*
```
Comment (mureytasroc) [inline .gitignore]: Why is the star necessary?

File: review/management/commands/pcrbackup.py (hunk 1 above)
Comment (davish) [reply review/management/commands/pcrbackup.py]: I could totally be wrong on this, but I think the `y/N` convention is just conveying that `N` is the default value if you press enter: it's not actually enforcing that `y` needs to be lowercase and `N` needs to be uppercase

File: .gitignore (hunk 2 above)
Comment (davish) [reply .gitignore]: Mainly so we can have multiple backup directories. On my machine, `pcr-backup` contains all PCR data up until 2018A -- I made a new directory called `pcr-backup-semester` to test this feature out, wanted to make sure it was also in the gitignore

File: review/management/commands/pcrbackup.py (hunk 1 above)
Comment (mureytasroc) [reply review/management/commands/pcrbackup.py]: Oh ok that makes sense

File: .gitignore (hunk 2 above)
Comment (mureytasroc) [reply .gitignore]: Makes sense

**Pull request merged**